services/blockchain-connector/methods/tests/decoder.js

```diff
@@ -36,10 +36,10 @@ module.exports = [
 	{
 		name: 'decodeTransactionSerialized',
 		params: {
-			encodedTransaction: { optional: false, type: 'string' },
+			transaction: { optional: false, type: 'object' },
```

Comment:
The params do not justify the method name. Please consider creating a separate method or renaming the method to be more appropriate?

Reply:
refactoring endpoint name

Reply:
Applied.